topics/go/concurrency/channels/example1/example1.go

```diff
@@ -118,12 +131,12 @@ func poolingUnlimited() {
 	fmt.Println("-------------------------------------------------------------")
 }
 
-// poolingLimited: Think about being a manager and hiring a team of employees.
+// fanOutLimited: Think about being a manager and hiring a team of employees.
```

Comment:
Description is wrong and this is not in the right order as the ordering in main.